### DISCUSSION TITLE
[v5] flashMessages returns {values: null} if form data is too large

### DISCUSSION THREAD

**Discussion by leocabeza** [adonisjs/core#4128] [Help] [answered]
# Problem: 

When having a lot of form data to submit, flashMessages shows as null

# To reproduce:

1. yarn
2. yarn dev
3. Go to http://localhost:3333/orders/create
4. Without filling anything, click "Guardar" button at the bottom of the form.
5. Below the button, you can see flashMessages shows as:
```json
{
  "values": null
}
```

# What I tried: 
* Increasing limits inside config/bodyparser.ts

# Repo & steps to reproduce:
https://github.com/leocabeza/adonis-error

**Comment by leocabeza** [comment]
This might be related to https://github.com/adonisjs/core/discussions/2488?

**Comment by McSneaky** [comment]
Most likely, because max cookie size is 4096bytes, which includes all cookie settings for all cookies combined: https://chromestatus.com/feature/4946713618939904

It's a browser implementation thing, can't do anything about it :|

**Reply by leocabeza** [reply, marked as answer]
Awesome, changing my session driver to 'redis' solved it for me, just as the suggestion in this issue: https://github.com/adonisjs/core/discussions/2488
